config/vsc_hortense.py

```diff
@@ -28,7 +28,11 @@
 from eessi.testsuite.constants import *  # noqa: F403
 
 # Note that we rely on the SBATCH_ACCOUNT environment variable to be specified
-hortense_access = ['--export=NONE', '--get-user-env=60L']
+# From ReFrame 4.8.1 we can no longer rely on SBATCH_ACCOUNT completely
+# ReFrame unsets all `SBATCH_*` evironment variables before running `sbatch`
+# See https://github.com/reframe-hpc/reframe/issues/3422
+sbatch_account = os.getenv('SBATCH_ACCOUNT', None)
+hortense_access = [f'-A {sbatch_account}', '--export=NONE', '--get-user-env=60L']
```

Comment:
This should be the 100% equivalent:

```suggestion
hortense_access = ['--export=NONE', '--get-user-env=60L']
if sbatch_account:
    hortense_access.append(f'-A {sbatch_account}'
```